playbooks/support/responding.md

```diff
@@ -11,6 +11,14 @@ respond to these issues.
   These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
   immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
   capacity in response to load.
+- Monitor automated uptime incidents created by OpsGenie.
+  - For high-criticality systems, OpsGenie will open incidents if
+    [Pingdom](https://my.pingdom.com/app/newchecks/checks) uptime checks are failing. Alerts will automatically
+    close if resolved.
+  - All engineers should have the [OpsGenie mobile app](http://itunes.apple.com/us/app/opsgenie/id528590328?mt=8)
+    installed and be logged in.
+  - Select your preferences for push notifications, text messages, and/or phone calls for alerting in your
+    [OpsGenie user notification settings](https://artsy.app.opsgenie.com/settings/user/notification).
 
```

Comment:
Perhaps a note about acknowledging ( acking ) the alert - we want to get folks in this habit before we define clear response SLAs and add escalation policies